src/bundle.c

```diff
@@ -265,6 +265,90 @@ gboolean create_bundle(const gchar *bundlename, const gchar *contentdir, GError
 	return res;
 }
 
+static gboolean truncate_bundle(const gchar *inpath, const gchar *outpath, gsize size, GError **error) {
```

Comment:
This just copies the first size bytes from inpath to outpath. Is there no simpler way?